client-java/src/main/java/uk/gov/gchq/palisade/client/internal/resource/ErrorMessage.java

```diff
@@ -32,30 +27,29 @@
  * which the class is created is determined by the {@code ImmutableStyle}. This
  * class is also compatible with Jackson.
  *
- * @see ResourceClient
+ * @see WebSocketClient
  * @see "https://immutables.github.io/style.html"
  * @since 0.5.0
  */
 @Value.Immutable
-@ImmutableStyle
-@JsonDeserialize(as = Error.class)
-@JsonSerialize(as = Error.class)
-public interface IError extends Serializable {
+@JsonDeserialize(as = ImmutableErrorMessage.class)
+@JsonSerialize(as = ImmutableErrorMessage.class)
```

Comment:
That's there so that testing is far easier. It allows creating the object and then serialising it to JSON without having to manually create and embed the JSON in the test which is error prone with all the escaping required. This would be so much easy with multi-line strings.